e2e/rbd.go

```diff
@@ -275,6 +283,14 @@ var _ = Describe("RBD", func() {
 			if err != nil {
 				framework.Failf("failed to create node label: %v", err)
 			}
+			err = createNodeLabel(f, crushLocationRegionLabel, crushLocationRegionValue)
+			if err != nil {
+				framework.Failf("failed to create node label: %v", err)
+			}
+			err = createNodeLabel(f, crushLocationZoneLabel, crushLocationZoneValue)
+			if err != nil {
+				framework.Failf("failed to create node label: %v", err)
+			}
```

Comment:
lets enhance CreateNodeLable to take map[string]string and set the label once (not as part of this PR)

Reply:
Created a issue - #4146